src/Aacotroneo/Saml2/Saml2Auth.php

```diff
@@ -99,8 +100,13 @@ function sls()
     {
         $auth = $this->auth;
 
-        $keep_local_session = true; //we don't touch session here
-        $auth->processSLO($keep_local_session);
+        // destroy the local session by firing the Logout event
+        $keep_local_session = false;
+        $session_callback = function () {
+            \Event::fire(new Saml2LogoutEvent());
+        };
+
+        $auth->processSLO($keep_local_session, null, false, $session_callback);
```

Comment:
the https://github.com/onelogin/php-saml/pull/93/files is a bit confusing. If you keep your session, and provide a callback.. It's not called.
And if you callback doesnt destroy session, session is not destroyed (but you provide a keep_local_session = false).
But well, for what we needed it will work. We should change the '$keep_local_session' name to reflect that we are not destroying session here. Now it's really a '$destroy_local_session = false; //use our callback'